transiter.go

```diff
@@ -196,6 +196,11 @@ func main() {
 						Usage: "Maximum number of vehicles that will be returned in a single list vehicles request. Specifying a value <= 0 will disable the limit.",
 						Value: 100,
 					},
+					&cli.Int64Flag{
```

Comment:
Would it be simpler to just have a max-entities-per-request flag and use the same value for vehicles, shapes, stops, etc?

Reply:
Awesome thank you very much!